src/tes/worker/engine.go

```diff
@@ -25,14 +26,14 @@ type Engine interface {
 
 // engine is the internal implementation of a docker job engine.
 type engine struct {
-	conf Config
+	conf tes.Worker
```

Comment:
I'm being picky here, but `tes.Worker` doesn't read like a config type. So I'm contradicting my earlier comment about ServerConfig.

Possibly the most idiomatic Go approach would be to move the config stuff to a `config` package, then all config structs are referenced e.g. `config.Worker`